### ISSUE TITLE
How to suppress irrelevant warnings about AWS credentials

### ISSUE THREAD

**Issue by robvanderleek** [ashiina/lambda-local#224]
I'm using this library via the [Netlify CLI](https://github.com/netlify/cli) to locally run serverless Netlify functions.
In the console I'm seeing these two warning lines for every function invocation:

```
warning Using both auth systems: aws_access_key/id and secret_access_token !
warning Using both auth systems: aws_access_key/id and secret_access_token !
```

I've tracked down the [origin of these warnings](https://github.com/ashiina/lambda-local/blob/4409022d20b81cb4ff445cf1f8fe3ea7a0736cd2/src/lib/utils.ts#L127) and my AWS credentials file indeed contains entries for `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_SESSION_TOKEN ` for the default profile. However, I'm using these credentials to access AWS and they are not relevant for locally running the Netlify functions.

Is there a way to suppress these warnings?

**Comment by gpotter2** [comment]
Hi, you're right: it shouldn't be on us to display those warnings. I'll drop them

**Comment by gpotter2** [comment]
Fixed in 2.0.3 !

**Comment by robvanderleek** [comment]
Wow, that's fast, nice! 😄 

I'll try to open a PR in Netlify CLI to [update to 2.0.3](https://github.com/netlify/cli/blob/12ef199e51d196b143ae14a2e6c3f3472022c0d1/package.json#L286)
